fitnessapp/application.py

```diff
@@ -1,6 +1,6 @@
 
 """
-Copyright (c) 2023 Rajat Chandak, Shubham Saboo, Vibhav Deo, Chinmay Nayak
+Copyright (c) 2023 Arkoh Lawrence, Lingjun Liu, Habib Mohammed
```

Comment:
Could you change the year to 2024?
Could you check your name as well?

Reply:
@lingjunliu okay,  could you point the exact issue with my name?

Reply:
Lawrence is your first name, right? The order of your name